.circleci/config.yml

```diff
@@ -150,6 +150,28 @@ jobs:
           name: upload
           command: bash <(curl -s https://codecov.io/bash) -f coverage.txt
 
+  localnet:
+    working_directory: /home/circleci/.go_workspace/src/github.com/cosmos/cosmos-sdk
+    machine:
+      image: circleci/classic:latest
+    environment:
+      GOBIN: /home/circleci/.go_workspace/bin
+      GOPATH: /home/circleci/.go_workspace/
+      GOOS: linux
+      GOARCH: amd64
+    parallelism: 1
+    steps:
+      - checkout
+      - run:
+          name: run localnet and exit on failure
+          command: |
+            set -x
+            make get_tools
+            make get_vendor_deps
+            make build-linux
+            make localnet-start
+            docker run -it --network=host jackzampolin/gaiad-poll:latest ./poll.sh 40 5 10 localhost
```

Comment:
I still am not a fan of this. You're right that it's just in CI, which we don't give a trusted role (presently), but it's hard to debug - now if something breaks where do I go to figure out what's in your Docker image?

Tried going to https://hub.docker.com/r/jackzampolin/gaiad-poll/ but that doesn't appear to have a Dockerfile

Can we just run the poll script natively in CI?